templates/events-my-events.php

```diff
@@ -18,10 +18,68 @@
 $event_page_title = __( 'My Events', 'gp-translation-events' );
 gp_tmpl_load( 'events-header', get_defined_vars(), __DIR__ );
 ?>
-
+<div class="events-links-to-anchors">
+	<ul>
+		<?php if ( ! empty( $events_i_am_or_will_attend_query->events ) ) : ?>
+			<li><a href="#events-i-am-or-will-attend"><?php esc_html_e( 'Events I am or will be attending', 'gp-translation-events' ); ?></a></li>
+		<?php endif; ?>
+		<?php if ( ! empty( $events_i_host_query->events ) ) : ?>
+			<li><a href="#events-i-host"><?php esc_html_e( 'Events I host', 'gp-translation-events' ); ?></a></li>
+		<?php endif; ?>
+		<?php if ( ! empty( $events_i_created_query->events ) ) : ?>
+			<li><a href="#events-i-created"><?php esc_html_e( 'Events I have created', 'gp-translation-events' ); ?></a></li>
+		<?php endif; ?>
+		<?php if ( ! empty( $events_i_attended_query->events ) ) : ?>
+			<li><a href="#events-i-attended"><?php esc_html_e( 'Events I attended', 'gp-translation-events' ); ?></a></li>
+		<?php endif; ?>
+		</ul>
+</div>
 <div class="event-page-wrapper">
+	<?php if ( ! empty( $events_i_am_or_will_attend_query->events ) ) : ?>
+		<h2 id="events-i-am-or-will-attend"><?php esc_html_e( 'Events I am or will be attending', 'gp-translation-events' ); ?> </h2>
+		<ul>
+		<?php
+		foreach ( $events_i_am_or_will_attend_query->events as $event ) :
+			$stats_calculator = new Stats_Calculator();
+			$has_stats        = $stats_calculator->event_has_stats( $event->id() );
+			?>
+			<li class="event-list-item">
+				<a class="event-link-<?php echo esc_attr( $event->status() ); ?>" href="<?php echo esc_url( Urls::event_details( $event->id() ) ); ?>"><?php echo esc_html( $event->title() ); ?></a>
+				<?php if ( ! $event->end()->is_in_the_past() && ! $has_stats ) : ?>
```

Comment:
```suggestion
				<?php if ( current_user_can( 'edit_translation_event', $event->id() ) : ?>
```

Reply:
Good catch!!!! I have updated the logic here and in another 2 places!